src/System.Management.Automation/engine/remoting/fanin/OutOfProcTransportManager.cs

```diff
@@ -773,17 +771,25 @@ protected void HandleOutputDataReceived(string data)
         {
             try
             {
+                if (string.IsNullOrEmpty(data))
```

Comment:
Minor comment.  The null check does not need to be within the try/catch.

Reply:
Done.